src/redux/migrations.ts

```diff
@@ -1824,4 +1824,15 @@ export const migrations = {
   }),
   220: (state: any) => state,
   221: (state: any) => state,
+  222: (state: any) => ({
+    ...state,
+    recipients: {
+      ..._.omit(state.recipients, 'valoraRecipientCache'),
+      appRecipientCache: state.valoraRecipientCache,
+    },
+    keylessBackup: {
```

Comment:
this isn't persisted, so you don't need a migration

Reply:
should we remove this part then?